pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1054,6 +1054,49 @@ class Build {
             }
         }
     }
+
+    // Kick off the sign_temurin_jsf job to sign the SBOM
```

Comment:
From the JSON vs XML discussion we've just had, we probably ought to call this job something slightly different... maybe simply sign_SBOM...

For the moment, we can have the XML discussion before we decide.

Reply:
using JSON signing for the moment